examples/example_05_dots_segmentation.py

```diff
@@ -0,0 +1,165 @@
+"""
+DOTS white matter segmentation
+==============================
+
+This example shows how to perform white matter segmentation using Diffusion
+Oriented Tract Segmentation (DOTS) algorithm [1]_:
+
+1. Downloading DOTS atlas prior informationg using
+    :func:`nighres.data.download_DOTS_atlas`
+2. Downloading an example DTI data set using
+    :func:`nighres.data.download_DTI_2mm`
+3. Segmenting white matter in DTI images into major tracts using 
+    :func:`nighres.brain.dots_segmentation` [1]_
```

Comment:
You can only use the footnote ([1]_) once unfortunately, since you use it just a few lines above I would just remove it here.